.gitignore

```diff
@@ -1,7 +1,9 @@
+*.Rcheck/
```

Comment:
These are left behind by `R CMD check` and `R CMD build`, respectively, so I added them to .gitignore